tinygrad/graph.py

```diff
@@ -14,7 +14,7 @@
 
 # **** debugging and graphing ****
 
-G = nx.DiGraph() if nx is not None else None
+G = nx.DiGraph()
```

Comment:
if networkx is not installed, this will fail with `AttributeError: 'NoneType' object has no attribute 'DiGraph'`

Reply:
If setup doesnt work we have a different problem also this is handled above.